### PR TITLE
fix(wasm): automatically refresh wallet nonce

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
-

### Fixes
- fixed wallet nonce issue. and added automatically refresh wallet nonce in background worker

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/1987

### CODE REVIEW

File: wasmsdk/wallet.go [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+	c := client.GetClient()
+	c.ClientID = clientID
+	c.ClientKey = publicKey
+
+	w := &zcncrypto.Wallet{
+		ClientID:  clientID,
+		ClientKey: publicKey,
+		Keys: []zcncrypto.KeyPair{
+			{
+				PrivateKey: privateKey,
```
Comment (peterlimg) [inline wasmsdk/wallet.go:32]: Actually it's not safe to have raw private key in memory all the time, theoretically we should only load it when need to sign transaction, and then wipe it immediately. Public key should be enough to get wallet balance. If this setWallet is also being used by other services, then we need to fix the private key in memory issue with a new PR.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: wasmsdk/wallet.go (hunk 1 above)
Comment (cnlangzi) [reply wasmsdk/wallet.go:32]: @peterlimg  in web-apps, wallet is already saved in a global js object. wasm wallet is a copy of the js object. and wasm sdk doesn't know which js object it is in memory. so wasm sdk can't load it on demand.  @ranajahanzaib  do you know other option?

Comment (cnlangzi) [reply wasmsdk/wallet.go:32]: in web-apps, private key is required to sign almost requests.

Comment (peterlimg) [reply wasmsdk/wallet.go:32]: okay, we may need to revise the design later, can you create an issue to track it, I'll merge this one first.

**Review by peterlimg — APPROVED** [review]
lgtm

File: wasmsdk/wallet.go (hunk 1 above)
Comment (cnlangzi) [reply wasmsdk/wallet.go:32]: @peterlimg  raise an issue to discuss it on https://github.com/0chain/gosdk/issues/720

**Pull request merged**